### PR TITLE
ADR 61: Displaying Stats in UTC Date Ranges

### PR DESCRIPTION
**Pull request by goplayoutside3**
## Package
lib-user
lib-content

## Linked Issue and/or Talk Post
Toward: https://github.com/zooniverse/front-end-monorepo/issues/6400

## Describe your changes
- ADR 61 describes the need for clearly labeled UI components that display stats data and three scenarios where volunteers could be confused by stats displayed in dates other than their local timezone.
-  I added a visual label below the x-axis on lib-user's BarChart. This BarChart is only ever used to display user stats or groups stats, so we can assume its x-axis unit is a date range in UTC.
- I also added a brief question + answer to the FAQ page for volunteer and zoo team reference.

## How to Review
- Please read through the ADR and comment if anything should be added or revised.
- Run app-root locally and visit your personal stats page and/or a group stats page. The bar chart has a label below the x-axis indicating the date range is UTC.
- The FAQ page has a new question + answer.
- Refactoring app-project's YourStats component will be a future PR.

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General
- [x] Tests are passing locally and on Github
- [x] Documentation is up to date and changelog has been updated if appropriate
- [x] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected
- [x] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari (Use Browserstack account as needed)
- [ ] FEM works in a mobile browser

### CODE REVIEW

**Comment by goplayoutside3** [comment]
@seanmiller26 we already chatted on Slack about the x-axis label. Here are the screenshots again for reference.
desktop
mobile


Here's a screenshot of the FAQ question + answer. The text is easily revised before or after this PR is merged if needed.
faq

**Comment by goplayoutside3** [comment]
@mcbouslog I'm requesting your code review because your familiarity with the BarChart.

@kieftrav and @shaunanoordin I've tagged `@frontend` for review too because of the ADR. It's a good idea to read through it for reference in case you're answer questions from volunteers about stats or debugging stats components in the future.

File: docs/arch/adr-61.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+Launch of the new homepages, user stats pages, and group stats pages have put a spotlight on our stats service. The new features allow users to see their classification counts and time spent increase after every contribution to a project.
+
+Classification metadata is stored in the database with UTC timestamps, and the stats service, ERAS, aggregates classification counts into buckets by UTC date ranges. However, not all UI components have clear labels indicating a UTC data range, and volunteers could encounter three unintuitive (but expected) scenarios:
+
+### When a volunteer's local timezone is behind UTC
+
+Example: A volunteer submits classifications at 9pm Chicago, but it's 3am UTC.
+
+Classification metadata is stored in our database with UTC timestamps. If a volunteer's local timezone is Oct 31, but the UTC date is Nov 1 then their classifications are put into the time bucket for Nov 1. In a bar chart UI componenent that displays "Last 7 Days", the volunteer sees Oct 26 - Nov 1. As this volunteer submits classifications, their classification count will increase in the bar for Nov 1 (not their local time of 9pm Oct 31).
```
Comment (mcbouslog) [inline docs/arch/adr-61.md:17]:
```suggestion
Classification metadata is stored in our database with UTC timestamps. If a volunteer's local timezone is Oct 31, but the UTC date is Nov 1 then their classifications are put into the time bucket for Nov 1. In a bar chart UI component that displays "Last 7 Days", the volunteer sees Oct 26 - Nov 1. As this volunteer submits classifications, their classification count will increase in the bar for Nov 1 (not their local time of 9pm Oct 31).
```

File: docs/arch/adr-61.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+### When a volunteer's local timezone is ahead of UTC
+
+Example: A volunteer submits classifications at 8am Korea, but it's 11pm UTC.
+
+If a volunteer's local timezone is Nov 2, but the UTC date is Nov 1, then their classifications are still put into the time bucket for Nov 1. In a bar chart UI componenent that displays "Last 7 Days", the volunteer sees Oct 26 - Nov 1. As this volunteer submits classifications, their classification count will increase in the bar for Nov 1 (not their local time of 8am Nov 2). This volunteer won't see a bar for Nov 2 on the chart.
```
Comment (mcbouslog) [inline docs/arch/adr-61.md:23]:
```suggestion
If a volunteer's local timezone is Nov 2, but the UTC date is Nov 1, then their classifications are still put into the time bucket for Nov 1. In a bar chart UI component that displays "Last 7 Days", the volunteer sees Oct 26 - Nov 1. As this volunteer submits classifications, their classification count will increase in the bar for Nov 1 (not their local time of 8am Nov 2). This volunteer won't see a bar for Nov 2 on the chart.
```

**Review by mcbouslog — APPROVED** [review]
ADR breaks down the topic well, I really appreciate/like the examples.

FAQ item is helpful reference 👍 .
BarChart content looks good across screens/theme.

Noting default end date (i.e. here in [UserStats](https://github.com/zooniverse/front-end-monorepo/blob/master/packages/app-root/src/app/users/%5Blogin%5D/stats/UserStatsContainer.js#L38) or [GroupStats](https://github.com/zooniverse/front-end-monorepo/blob/master/packages/app-root/src/app/groups/%5BgroupId%5D/GroupStatsContainer.js#L40)) will be UTC date, consistent with this ADR.

**Pull request merged**